tests/e2e/performance/test_full_cluster_health.py

```diff
@@ -0,0 +1,284 @@
+"""
+Test to verify cluster health/stability when it's full (80%)
+"""
+import logging
+import time
+from ocs_ci.ocs.cluster import CephCluster, get_percent_used_capacity
+from ocs_ci.ocs import constants, benchmark_operator, ocp, defaults
+from ocs_ci.ocs.perftests import PASTest
+from ocs_ci.helpers.performance_lib import run_command
+from ocs_ci.utility import templating
+from ocs_ci.helpers.helpers import get_full_test_logs_path
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.disruptive_operations import osd_node_reboot
+from ocs_ci.ocs.node import wait_for_nodes_status
+
+logger = logging.getLogger(__name__)
+
+
+class TestFullClusterHealth(PASTest):
+    """
+    Test Cluster health when storage is 80%
+    """
+
+    percent_to_fill = 78.0
+    ceph_cluster = CephCluster()
+    nodes = None
```

Comment:
you need to move lines 24-26 into the setup function with the prefix of `self.`

Reply:
Done